relay-redis/src/real.rs

```diff
@@ -382,3 +396,154 @@ pub struct Stats {
     /// The number of idle connections.
     pub idle_connections: u32,
 }
+
+/// [`bb8::ManageConnection`] for a async redis cluster.
+pub struct RedisClusterConnectionManager {
+    client: ClusterClient,
+}
+
+impl RedisClusterConnectionManager {
+    /// Creates a new [`bb8::ManageConnection`] with the provided [`ClusterClient`]
+    pub fn new(client: ClusterClient) -> Self {
+        Self { client }
+    }
+}
+
+impl bb8::ManageConnection for RedisClusterConnectionManager {
+    type Connection = ClusterConnection;
+    type Error = redis::RedisError;
+
+    fn connect<'life0, 'async_trait>(
```

Comment:
We can just use `async_trait` here, it's in the dependency tree anyways.

Reply:
Seems like I misunderstood what the [dev guide](https://develop.sentry.dev/engineering-practices/rust/#async-traits) says about `async_trait`. I thought it suggests to not use the crate, but after reading it again it does not mention this at all.

I will update accordingly

Reply:
Note, that link is also slightly outdated, Rust traits can now be `async fn`.